README.md

```diff
@@ -106,11 +107,11 @@ Value is one of:
 Thus, if you have a node whose data is given by:
 
 ````JavaScript
-{text: "My Node", data: {price: "$10"}}
+{text: "My Node", data: {price: 10}}
```

Comment:
Ditto. Really like how it shows the difference between `data` (what I called "raw" in the earlier discussion) and `presentation`.